packages/react-components/src/components/AttachmentCard.test.tsx

```diff
@@ -30,7 +30,13 @@ describe('AttachmentCard should be rendered properly', () => {
 
   it('should render the component with action icon', () => {
     renderAttachmentCardWithDefaults({
-      actionIcon: <Icon iconName="CancelFileUpload" />
+      menuActions: [
+        {
+          name: 'Cancel',
+          icon: <Icon iconName="CancelFileUpload" />,
```

Comment:
migrate to Fluent icon?

Reply:
this is fluent icon with a wrapper, the actual icon is defined in our theming here: 
<img width="856" alt="image" src="https://github.com/Azure/communication-ui-library/assets/109105353/c1e3de24-2750-49cc-b8da-0183694535e2">